cse_site/ubs_project/templates/ubs_project/exchange_create.html

```diff
@@ -33,16 +29,22 @@
 
 {% block js %}
     <!-- load editor js -->
-    <script src="{% static 'editor/editormd.js' %}"></script>
-    <script src="{% static 'editor/languages/en.js' %}"></script>
+<!--    rich text editor-->
```

Comment:
Might be a good idea to put this text editor in a separate javascript file that can be included with just a script src?